http/query_params.go

```diff
@@ -123,6 +123,16 @@ func (qp QueryParams) Key() string {
 	return qp["key"]
 }
 
+// SqlTimeout returns the value of the key named "sql_timeout_seconds".
+func (qp QueryParams) SqlTimeout(def time.Duration) time.Duration {
```

Comment:
Go Style says to call this "SQLTimeout"

Reply:
Also, add a check at line 29 for the validity of this variable, if exists, then you can parse directly here.

Reply:
I'm also torn about the use of "sql". I think "db_timeout" is better.

Reply:
Thank you; I was quite unsure about the naming here. I wanted something that conveys that this is a timeout per query, not per transaction or per database connection.